environment-dev.yml

```diff
@@ -22,6 +22,7 @@ dependencies:
   - spotpy
   - stackstac
   - statsmodels
+  - ravenpy
```

Comment:
On dirait que ça tente de se bâtir sur une vieille version de `ravenpy`.
```suggestion
  - ravenpy >=0.14.0
```

Reply:
@Zeitsperre I could create an environment with `ravenpy>=0.14.0` and `python v3.11.x`, but not `3.12`. 

In a fresh environment create from the `environment-dev.yml` on `main`, I get:
![image](https://github.com/hydrologie/xhydro/assets/38501935/31d7c82a-9a56-4df5-9cf1-bc6410238593)

Reply:
See: https://github.com/conda-forge/ravenpy-feedstock/pull/37

Reply:
This is fixed with v0.14.0-build1.